README.md

```diff
@@ -267,6 +271,32 @@ Call the `intersectionMockInstance` method with an element, to get the (mocked)
 `IntersectionObserver` instance. You can use this to spy on the `observe` and
 `unobserve` methods.
 
+#### Global Intersection Observer Behavior
+
+##### Use Fallback
+
+You can create a [Jest setup file](https://jestjs.io/docs/configuration#setupfiles-array) that leverages the [unsupported fallback](https://github.com/thebuilder/react-intersection-observer#unsupported-fallback) to that tests encountering Intersection Observer all behave the same way, like so:
```

Comment:
```suggestion
You can create a [Jest setup file](https://jestjs.io/docs/configuration#setupfiles-array) that leverages the [unsupported fallback](https://github.com/thebuilder/react-intersection-observer#unsupported-fallback) option. In this case, you only mock the IntersectionObserver in test files were you actively import `react-intersection-observer/test-utils`:
```

Reply:
Last part of the sentence is missing a word or two- maybe something more like this?